lib/pliny/log.rb

```diff
@@ -72,6 +72,14 @@ def stderr
       @stderr
     end
 
+    def scrubber=(s)
```

Comment:
What do you think about calling this `log_scrubber` instead? From the outside it provides more context than `Pliny.scrubber`.